commands/server.go

```diff
@@ -221,6 +223,8 @@ func setConfig() {
 	serverCmd.PersistentFlags().Int("ansible_connection_retries", 5, "Number of retries in case of Ansible SSH connection failure")
 	serverCmd.PersistentFlags().String("operation_remote_base_dir", ".yorc", "Name of the temporary directory used by Ansible on the nodes")
 	serverCmd.PersistentFlags().Bool("keep_operation_remote_path", config.DefaultKeepOperationRemotePath, "Define wether the path created to store artifacts on the nodes will be removed at the end of workflow executions.")
+	serverCmd.PersistentFlags().Bool("ansible_archive_artifacts", config.DefaultArchiveArtifacts, "Define wether artifacts should be archived before being copied on remote nodes.")
```

Comment:
Can we explicitly say that tar is required on remote hosts here?

Reply:
yes better to have it here as well, not just in the documentation

Reply:
We could install tar on remote hosts if not present and if the archive_artifacts option has been selected ?

Reply:
That's ok for me.